.github/workflows/_publish-container.yml

```diff
@@ -46,18 +43,5 @@ jobs:
       - name: Login to ACR
         run: az acr login --name ${{ vars.CONTAINER_REGISTRY_NAME }}
 
-      - name: Setup Docker Buildx
-        uses: docker/setup-buildx-action@v3
-
       - name: Build and push container image
-        working-directory: ${{ inputs.docker_context }}
-        run: |
-          docker buildx create --use
-          docker buildx build \
-            --platform linux/amd64,linux/arm64 \
-            --build-arg VERSION=${{ inputs.version }} \
-            -t ${{ vars.CONTAINER_REGISTRY_NAME }}.azurecr.io/${{ inputs.image_name }}:${{ inputs.version }} \
-            -t ${{ vars.CONTAINER_REGISTRY_NAME }}.azurecr.io/${{ inputs.image_name }}:latest \
-            -f ${{ inputs.docker_file }} \
-            --push .
-          docker buildx rm
+        run: dotnet publish ${{ inputs.api_location }} -t:PublishContainer -r linux-x64 -p ContainerImageTags='"${{ vars.CONTAINER_REGISTRY_NAME }}.azurecr.io/${{ inputs.image_name }}:${{ inputs.version }};${{ vars.CONTAINER_REGISTRY_NAME }}.azurecr.io/${{ inputs.image_name }}:latest"'
```

Comment:
Can we also publish the `linux/arm64`? so it can run on macOS?

Reply:
I was not able to find any way to have the image being also available for `linux/arm`.